src/main/java/edu/harvard/iq/dataverse/pidproviders/PidUtil.java

```diff
@@ -83,26 +83,4 @@ private static String acceptOnlyDoi(String persistentId) {
         }
         return globalId.getAuthority() + "/" + globalId.getIdentifier();
     }
-
-    /**
-     * Deletes the DOI from DataCite if it can. Returns 204 if PID was deleted
-     * (only possible for "draft" DOIs), 405 (method not allowed) if the DOI
-     * wasn't deleted (because it's in "findable" state, for example, 404 if the
-     * DOI wasn't found, and possibly other status codes such as 500 if DataCite
-     * is down.
-     */
-    public static int deleteDoi(String persistentId, String baseUrl, String username, String password) throws IOException {
```

Comment:
The test in PidUtilTest that calls this method should be deleted:

```
-------------------------------------------------------------
COMPILATION ERROR : 
-------------------------------------------------------------
edu/harvard/iq/dataverse/pidproviders/PidUtilTest.java:[35,29] cannot find symbol
  symbol:   method deleteDoi(java.lang.String,java.lang.String,java.lang.String,java.lang.String)
  location: class edu.harvard.iq.dataverse.pidproviders.PidUtil
1 error
```

See also https://travis-ci.org/github/IQSS/dataverse/builds/710110797

Reply:
Fixed.